python/_tskitmodule.c

```diff
@@ -4347,15 +4347,18 @@ TableCollection_union(TableCollection *self, PyObject *args, PyObject *kwds)
     npy_intp *shape;
     tsk_flags_t options = 0;
     int check_shared = true;
+    int all_edges = false;
+    int all_mutations = false;
     int add_populations = true;
-    static char *kwlist[] = { "other", "other_node_mapping", "check_shared_equality",
-        "add_populations", NULL };
+    static char *kwlist[] = { "other", "other_node_mapping", "all_edges",
```

Comment:
Although it is internal code, best practice is to add them at the end. It's no different but avoid breaking if anyone is using it.